packages/material-ui-system/src/index.d.ts

```diff
@@ -205,6 +205,7 @@ export function style<PropKey extends string, Theme extends object>(
 ): StyleFunction<{ [K in PropKey]?: unknown } & { theme: Theme }>;
 
 // typography.js
+export const typographyFn: SimpleStyleFunction<'typography'>;
```

Comment:
Not sure what is the best name for this function as we are already exporting `typography` from the system :\

Reply:
`typographyVariant`?

Reply:
Definitely sounds better than `typograpohyFn` :))) Will rename it